applications/colonoscopy_segmentation/Dockerfile

```
@@ -0,0 +1,54 @@
+# syntax=docker/dockerfile:1
+
+# SPDX-FileCopyrightText: Copyright (c) 2022-2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+############################################################
+# Base image
+############################################################
+
+
+FROM nvcr.io/nvidia/cuda:12.9.0-base-ubuntu24.04
```

nitpick: no need for that comment on top if you don't have over layers. also lots of empty lines

```suggestion

FROM nvcr.io/nvidia/cuda:12.9.0-base-ubuntu24.04
```